### ISSUE TITLE
False-Positive E5903 on repeated fields

### ISSUE THREAD

**Issue by pashatyl** [nelfin/pylint-protobuf#43]
Given:
```proto
message TestClass {
  repeated int32 scores = 1;
}
```

```python
class TestClass(google.protobuf.message.Message):
    DESCRIPTOR: google.protobuf.descriptor.Descriptor = ...
    SCORES_FIELD_NUMBER: builtins.int
    scores: google.protobuf.internal.containers.RepeatedScalarFieldContainer[builtins.int] = ...

    def __init__(self,
        *,
        scores : typing.Optional[typing.Iterable[builtins.int]] = ...,
        ) -> None: ...
    def ClearField(self, field_name: typing_extensions.Literal[u"scores",b"scores"]) -> None: ...
global___TestClass = TestClass
```

```python
TestClass(scores=[1,2,3])
```

Warn but shouldn't:
```
E5903: Field "TestClass.scores" is of type 'int' and value <class 'list'> will raise TypeError at runtime (protobuf-type-error)
```

**Comment by nelfin** [comment]
Thanks for the report. Can you see if the latest changes in `pip install git+https://github.com/nelfin/pylint-protobuf@5117c01dbe6f9244f8f1b7b7e69bc15b6d22613f` address your issue?

**Comment by pashatyl** [comment]
Yes, it does! Pretty quick solution, thanks a lot!

**Comment by nelfin** [comment]
Cool, that's great to hear. I've released this fix along with the changes in v0.19.0 just now.
